_posts/en/newsletters/2021-05-26-newsletter.md

```diff
@@ -159,14 +159,21 @@ repo], [Hardware Wallet Interface (HWI)][hwi repo],
 [Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
 BOLTs][bolts repo].*
 
+- [Eclair #1810][] makes it mandatory for peers to signal and comply with the
+  `payment_secret` feature bit. The `payment_secret` feature thwarts the
+  de-anonymization attack whereby intermediary peers on a payment path can
```

Comment:
Do we have a good Optech reference to provide for this attack? @dongcarl

Reply:
Hmm... Looks like we have:
https://bitcoinops.org/en/newsletters/2019/12/04/#c-lightning-3259
https://bitcoinops.org/en/newsletters/2020/11/04/#c-lightning-4162

Reply:
Will update, sec